sdk/python/notebooks_config.ini

```diff
@@ -21,3 +21,11 @@ COMPUTE_NAMES = adv-energy-cluster-v2
 [automl-forecasting-task-energy-demand-advanced-mlflow]
 USE_FORECAST_REQUIREMENTS = 1
 COMPUTE_NAMES = energy-cluster-v2
+
+[automl-forecasting-recipe-univariate-experiment-settings]
+USE_FORECAST_REQUIREMENTS = 1
+COMPUTE_NAMES = recipe-cluster
```

Comment:
We do not need compute name hare, because the settings notebook does not use the compute.

Reply:
Need to discuss.